addons/mail/static/src/models/dropdown_menu_manager.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { many } from '@mail/model/model_field';
+
+registerModel({
+    name: 'DropdownMenuManager',
+    fields: {
+        // FIXME: dependent on implementation that uses insert order in relations!!
```

Comment:
Don't keep fixme in code, either do it or create a task.
I think in this case the comment is just obsolete.